### PR TITLE
changes char overdose to the boomer list

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

#48381 added a bunch of lines for monkey energy when that was literally the gimmick i used with char while monkey energy was supposed to be fuel for the grill
so now i update char overdose to just be the monkey energy overdose but at a lower overdose threshhold since its nicer

## Why It's Good For The Game

yes

## Changelog
:cl:
tweak: the char overdose threshhold is now lower, also the lines have been changed
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/food_reagents.dm [hunk 1]
```diff
@@ -729,13 +729,13 @@
 	color = "#C8C8C8"
 	taste_mult = 6
 	taste_description = "smoke"
-	overdose_threshold = 25
+	overdose_threshold = 20
 
-/datum/reagent/consumable/char/overdose_process(mob/living/carbon/M)
-	if(prob(10))
-		M.say(pick("I hate my wife.", "I just want to grill for God's sake.", "I wish I could just go on my lawnmower and cut the grass.", "Yep, Quake. That was a good game...", "Yeah, my PDA has wi-fi. A wife I hate."), forced = /datum/reagent/consumable/char)
+/datum/reagent/consumable/monkey_energy/overdose_process(mob/living/M)
```
Comment (TheChosenEvilOne) [inline code/modules/reagents/chemistry/reagents/food_reagents.dm]:
```suggestion
/datum/reagent/consumable/char/overdose_process(mob/living/M)
```

Comment (Fikou) [reply code/modules/reagents/chemistry/reagents/food_reagents.dm]: bada bim bada bum

File: code/modules/reagents/chemistry/reagents/food_reagents.dm [hunk 2]
```diff
@@ -729,13 +729,13 @@
...
 	taste_description = "smoke"
-	overdose_threshold = 25
+	overdose_threshold = 15
 
-/datum/reagent/consumable/char/overdose_process(mob/living/carbon/M)
-	if(prob(10))
-		M.say(pick("I hate my wife.", "I just want to grill for God's sake.", "I wish I could just go on my lawnmower and cut the grass.", "Yep, Quake. That was a good game...", "Yeah, my PDA has wi-fi. A wife I hate."), forced = /datum/reagent/consumable/char)
+/datum/reagent/consumable/char/overdose_process(mob/living/M)
+	if(prob(25))
+		M.say(pick_list_replacements(BOOMER_FILE, "boomer"), forced = /datum/reagent/consumable/monkey_energy)
```
Comment (kriskog) [inline code/modules/reagents/chemistry/reagents/food_reagents.dm]: forced = /datum/reagent/consumable/monkey_energy

Comment (optimumtact) [inline code/modules/reagents/chemistry/reagents/food_reagents.dm]: it's not monkey energy though is it, it's `/datum/reagent/consumable/char`

Comment (TheChosenEvilOne) [inline code/modules/reagents/chemistry/reagents/food_reagents.dm]: This too probably:
```suggestion
		M.say(pick_list_replacements(BOOMER_FILE, "boomer"), forced = /datum/reagent/consumable/char)
```

**Comment by kriskog** [comment]
Fastest reviewed PR in the west :nerd_face:

File: code/modules/reagents/chemistry/reagents/food_reagents.dm (hunk 2 above)
Comment (Fikou) [reply code/modules/reagents/chemistry/reagents/food_reagents.dm]: pee

Comment (Fikou) [reply code/modules/reagents/chemistry/reagents/food_reagents.dm]: poo

Comment (Fikou) [reply code/modules/reagents/chemistry/reagents/food_reagents.dm]: paa

**Comment by optimumtact** [comment]
Fastest reviewed PR in the west :nerd_face:

**Comment by Fikou** [comment]
Fastest reviewed PR in the west 🤓

**Comment by TheChosenEvilOne** [comment]
Fastest reviewed PR in the west

**Review by Cyberboss — APPROVED** [review]

**Comment by TheVekter** [comment]
Fastest reviewed PR in the west 🤓

**Comment by trollbreeder** [comment]
Fasted reviewed PR in the east 🤡

**Comment by HomocidlePancakes** [comment]
Fucking cringe

**Pull request merged**